boost-maven-plugin/src/main/java/io/openliberty/boost/docker/AbstractDockerMojo.java

```diff
@@ -71,7 +62,7 @@
     /**
      * Connect to Docker Daemon using HTTP proxy, if set.
      */
-    @Parameter(defaultValue = "true", property = "useProxy")
```

Comment:
You can remove `property = "useProxy"` since the variable is already named that.

Reply:
Doesn't removing the 'property' attribute of the annotation prevent you from using a -D override from the command line?   E.g. `mvn... -DuseProxy=false`

Reply:
You're right, didn't realize that. Ok, we can keep it then.